client/src/App.js

```diff
@@ -17,10 +18,12 @@ function App() {
         <Route path="/" element={<Home />} />
         <Route path="/:id" element={<CardInfo />} />
         <Route path="/form" element={<Form />} />
+        <Route path="/team-alpha" element={<TeamAlphaDetails />} />
+        <Route path="/search" element={<Search/> } />
       </Routes>
       <Footer />
     </Router>
-);
+  );
 }
 
 export default App;
```

Comment:
Thanks Saghar. Great job. Just a few things to look at update. Thank you for your hard work dear. x